avocado/core/resolver.py

```diff
@@ -189,6 +201,19 @@ def check_file(
     return True
 
 
+def get_file_assets(test_file_path):
+    """Gets asset files from the test file and its the ".data" directory
+
+    :param test_file_path: the filesystem location of the test file
+    :type test_file_path: str
+    :returns: list of tuples with (asset type, path)
+    """
+    file_assets = [(ReferenceResolutionAssetType.TEST_FILE, test_file_path)]
+    for data_file in glob.glob(f"{test_file_path}.data/*"):
```

Comment:
IIUIC this can be used only on Unix/Linux systems. Even that we don't test spawners on other platforms, IMO it is not necessary to add this constrain here. What about to use `os.walk()` here?

Reply:
Thanks for spotting this.  There's a violation of portability best practices here indeed, but it's a quick fix:

```diff
diff --git a/avocado/core/resolver.py b/avocado/core/resolver.py
index 8534ce23f..8dc1410e0 100644
--- a/avocado/core/resolver.py
+++ b/avocado/core/resolver.py
@@ -209,7 +209,7 @@ def get_file_assets(test_file_path):
     :returns: list of tuples with (asset type, path)
     """
     file_assets = [(ReferenceResolutionAssetType.TEST_FILE, test_file_path)]
-    for data_file in glob.glob(f"{test_file_path}.data/*"):
+    for data_file in glob.glob(os.path.join(f"{test_file_path}.data", "*")):
         file_assets.append((ReferenceResolutionAssetType.DATA_FILE, data_file))
     return file_assets
```